src/app/caesar-portal/common/services/group.service.ts

```diff
@@ -2,59 +2,41 @@ import { Injectable } from '@angular/core';
 import { Http, Response } from '@angular/http';
 import { environment } from '../../../../environments/environment';
 import { Group } from '../../common/models/group';
+import { User } from '../../common/models/user';
 import { Observable } from 'rxjs/Observable';
-import 'rxjs/add/operator/toPromise';
+import { Location } from '../../common/models/location';
+import { Subject } from 'rxjs/Subject';
 
 @Injectable()
 
 export class GroupService {
   group: Group = new Group();
+  groupCurrent = new Subject<Group>();
```

Comment:
it's better to use behavior subject for it - so you can keep the last one updated

Reply:
Yes, it's really, that we need! Thank you)